reference_codegen/generate.mjs

```diff
@@ -160,6 +160,12 @@ helpAll.scope_to_help_info[""].advanced.forEach((option) => {
     cachedir = option.default.split("/").slice(0, -2).join("/");
   }
 });
+// NB: The default changes depending on the environment, so hardcode.
+helpAll.scope_to_help_info[""].basic.forEach((option) => {
+  if (option.config_key === "dynamic_ui") {
```

Comment:
I'd suggest putting this `NB` next to the `if (...)` key, so that if/when we add more transformations to this function body, the comment is still attached to the right place. Probably no need to change now though, as long as we remember to move it in future if we need to.

```suggestion
helpAll.scope_to_help_info[""].basic.forEach((option) => {
  // NB: The default changes depending on the environment, so hardcode.
  if (option.config_key === "dynamic_ui") {
```